src/utils/blockchain/contracts.ts

```diff
@@ -220,6 +223,10 @@ export const contracts = {
     abi: VestingRegistryOriginAbi,
     blockNumber: 1218836,
   },
+  vestingRegistryBabelfish: {
+    address: '0xE402a46F372a461dD19ed34453253a0B5D1e0509',
+    abi: VestingRegistryAbi,
+  },
```

Comment:
// todo

Reply:
Actually, we don't need this one since we have LockedFund contract for that reason. I'll delete it in another PR because I need to make another adjustment as well.

Reply:
Removed in https://github.com/DistributedCollective/Sovryn-frontend/pull/1390